plugins/Goals/Visualizations/Goals.php

```diff
@@ -154,11 +154,57 @@ private function setPropertiesForEcommerceView()
         $this->requestConfig->filter_sort_column = 'goal_ecommerceOrder_revenue';
         $this->requestConfig->filter_sort_order = 'desc';
 
-        $this->config->columns_to_display = [
-            'label', 'nb_visits', 'goal_ecommerceOrder_nb_conversions', 'goal_ecommerceOrder_revenue',
-            'goal_ecommerceOrder_conversion_rate', 'goal_ecommerceOrder_avg_order_revenue', 'goal_ecommerceOrder_items',
-            'goal_ecommerceOrder_revenue_per_visit'
-        ];
+        switch ($this->displayType) {
+            case self::GOALS_DISPLAY_NORMAL:
+                $this->requestConfig->filter_sort_column = 'goal_ecommerceOrder_revenue';
+                $this->config->columns_to_display = [
+                    'label',
+                    'nb_visits',
+                    'goal_ecommerceOrder_nb_conversions',
+                    'goal_ecommerceOrder_revenue',
+                    'goal_ecommerceOrder_conversion_rate',
+                    'goal_ecommerceOrder_avg_order_revenue',
+                    'goal_ecommerceOrder_items',
```

Comment:
Looking into the code it seems like we do not calculate the items metric for pages. So we shouldn't show them here.
```suggestion
```